frontend/public/components/dashboard/dashboards-page/cluster-dashboard/getting-started/getting-started-section.scss

```
@@ -0,0 +1,12 @@
+.cluster-dashboard-getting-started-section {
```

nit: can we have as bleow?

```suggestion
.co-dashboard-getting-started-section {
```

---

Fixed

See https://github.com/openshift/console/compare/59068d40fba7a271e13b96803de8b5a9ed2c6ae6..fa1c708ab298c706378019e68660e990c6a39d34